README.md

```diff
@@ -27,6 +27,7 @@ The SAP BTP service operator is based on the [Kubernetes Operator pattern](https
 * [Credentials Rotation](#credentials-rotation)
 * [Multitenancy](#multitenancy)
 * [Troubleshooting and Support](#troubleshooting-and-support)
+* [Secret Formats](#secret-formats)
```

Comment:
Maybe Formats of 'Secret' Objects

This way it seems like formats are secret.

Reply:
done